ArduCopter/mode_acro.cpp

```diff
@@ -17,24 +16,32 @@ void ModeAcro::run()
     if (!motors->armed()) {
         // Motors should be Stopped
         motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::SHUT_DOWN);
-    } else if (copter.ap.throttle_zero && copter.air_mode != AirMode::AIRMODE_ENABLED) {
-        // Attempting to Land, if airmode is enabled only an actual landing will spool down the motors
+    } else if (copter.ap.throttle_zero
+               // throttle_zero is never true in air mode, but the motors should be allowed to go through ground idle
```

Comment:
minor formatting thing but I think these two lines of comments should go after the curly brace.  I see why it's here but we never put comments within the conditionals like this.

Reply:
Thanks, fixed